apps/studio/components/grid/SupabaseGrid.tsx

```diff
@@ -87,6 +87,16 @@ const SupabaseGridLayout = forwardRef<SupabaseGridRef, SupabaseGridProps>(
       }
     )
 
+    if (error?.error?.message.includes('does not exist')) {
```

Comment:
I'm not sure about this one because we have an 'error' within 'error', and it's of 'any' type. Is there a better way to handle it?
Full error message here: `column "name" does not exist`

Reply:
I think we can add `?` before includes to prevent it. but it will be better if we handle in `table-rows-query` part ?